### ISSUE TITLE
Link Launchpad task list to the frontend

### ISSUE THREAD

**Issue by nelsonec87** [Automattic/wp-calypso#77287]
Hook up the Customer Home Launchpad to the correct task list (yet to be created) on the back end.

*Since the backend and frontend work will be done in parallel, the frontend will use some existing task lists for testing.

Related to https://github.com/Automattic/wp-calypso/issues/77279 (creating the new Task list)

**Comment by sixhours** [comment]
Closed in https://github.com/Automattic/wp-calypso/pull/78147
